tests/lib/prepare.sh

```diff
@@ -95,7 +95,7 @@ setup_snapd_proxy() {
     mkdir -p /etc/systemd/system/snapd.service.d
     cat <<EOF > /etc/systemd/system/snapd.service.d/proxy.conf
 [Service]
-Environment="HTTPS_PROXY=$HTTPS_PROXY HTTP_PROXY=$HTTP_PROXY https_proxy=$HTTPS_PROXY http_proxy=$HTTP_PROXY NO_PROXY=$NO_PROXY no_proxy=$NO_PROXY"
+Environment=HTTPS_PROXY=$HTTPS_PROXY HTTP_PROXY=$HTTP_PROXY https_proxy=$HTTPS_PROXY http_proxy=$HTTP_PROXY NO_PROXY=$NO_PROXY no_proxy=$NO_PROXY
```

Comment:
for future, these don't have to be on a single line, at some point it may be easier to use format like this:
```suggestion
Environment=HTTPS_PROXY=$HTTPS_PROXY
Environment=HTTP_PROXY=$HTTP_PROXY
Environment=https_proxy=$HTTPS_PROXY
Environment=http_proxy=$HTTP_PROXY
Environment=NO_PROXY=$NO_PROXY
Environment=no_proxy=$NO_PROXY
```